accounts/views.py

```diff
@@ -73,11 +86,95 @@ class HomeView(LoginRequiredMixin, ListView):
     model = Tweet
     template_name = "accounts/home.html"
     paginate_by = 20
-    context_object_name = "tweets_list"
 
-    def get_queryset(self):
-        return Tweet.objects.all().select_related("user")
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context["tweets_list"] = Tweet.objects.select_related("user")
+        context["following_list"] = FriendShip.objects.select_related(
+            "follower", "following"
+        ).filter(follower=self.request.user)
+        return context
 
 
 class WelcomeView(TemplateView):
     template_name = "welcome/index.html"
+
+
+class FollowView(LoginRequiredMixin, TemplateView):
+    template_name = "accounts/follow.html"
+    model = FriendShip
+    context_object_name = "following_list"
+
+    def post(self, request, *args, **kwargs):
+        try:
+            follower = get_object_or_404(User, username=self.request.user.username)

```

Comment:
```suggestion
            follower = self.request.user
```
で良いと思います！（self.request.userとfollowerは結局同義なので）

Reply:
直しました